README.md

```diff
@@ -129,7 +129,7 @@ UI is ultimately driven by `js/utils/bbgmView.js`, a small UI layer I wrote on
 top of Knockout which is used by all the views in the `js/views` folder. Each
 view also has a corresponding HTML file in the `templates` folder. Adding a new
 page is kind of a bitch. You need to explicitly include the template file in
-`js/templates.js`, and explicitly include the view in `js/views.js`. Beyond
+`templates`, and explicitly include the view in `js/views`. Beyond
```

Comment:
Undo this change, the original text is correct. You need both the file in the templates folder and a reference to it in js/templates.js. Same thing with views.